app/models/proxy.rb

```diff
@@ -682,7 +680,7 @@ def call(attribute)
       begin
         uri = URI.parse(attribute_value)
         value = URI::Generic.new(uri.scheme, uri.userinfo, uri.host, uri.port, uri.registry, uri.path, uri.opaque, uri.query, uri.fragment).to_s
-        @model[attribute] = value
+        @model[attribute] = value unless @model[attribute] == value
```

Comment:
This afair is to avoid creating a proxy changes tracker when attribute is updated with the same value. See 2b9d5c4133a90af14a80468246c4a6795f897f80.

Hopefully we can refactor the class at some point.